components/src/components/embed/ComponentTree.tsx

```diff
@@ -66,6 +66,8 @@ export const ComponentTree = () => {
   }, [nodes]);
 
   if (error) {
+    console.error(error);
```

Comment:
We could be better about error handling but this is probably better served as a part of another ticket.

Reply:
Sure although it would help with the issue I'm truobleshooting with Gio now

Reply:
we talked about this live, keeping for now but agree we should revisit error handling